### PR TITLE
kubernetes: Merge scripts and styles directories

### PR DESCRIPTION
**Pull request by stefwalter**
These had a registry subdirectory with some duplication. Clean it up, and merge it, since much of it is used by both the kubernetes and registry views.

### CODE REVIEW

File: pkg/kubernetes/index.html [hunk 1]
```diff
@@ -51,14 +51,15 @@
...
-  <script src="scripts/registry/projects.js"></script>
+  <script src="scripts/layers.js"></script>
+  <script src="scripts/images.js"></script>
+  <script src="scripts/projects.js"></script>
   <!-- debug:end -->
 
 </head>
 <body ng-app='kubernetes' ng-controller="MainCtrl" ng-class="{'cards-pf': viewActive(null)}" hidden>
-      <div class="kube-sidebar" ng-if="!curtains">
+    <div ng-cloak ng-if="!curtains">
```
Comment (petervo) [inline pkg/kubernetes/index.html]: needs a matching </div> tag

Comment (stefwalter) [inline pkg/kubernetes/index.html]: Done.

**Comment by stefwalter** [comment]
There's a couple check-kubernetes and check-openshift test failures as well. They seem to be happening more broadldy than this pull request though.

**Comment by petervo** [comment]
The check-kubernetes failures were because of the div thing. The images one might be genuine.

**Comment by petervo** [comment]
Testing rebase in #4044

**Pull request closed without merging**